ui/src/AccountDropdown/InviteEditorsFormSection.tsx

```diff
@@ -89,22 +170,50 @@ function InviteEditorsFormSection({collapsed, currentSpace, currentUser, closeMo
         }
     }
 
-
     return (
         <form className="inviteEditorsForm form" onSubmit={inviteUsers}>
-            <label htmlFor="emailTextarea" className="inviteEditorsLabel">
-                People with this permission can edit
-            </label>
+            {!isExpanded && <span className={'inviteEditorsLabel'}>People with this permission can edit</span>}
             {isExpanded && (
                 <>
-                    <input
-                        id="emailTextarea"
-                        className="emailTextarea"
-                        placeholder="cdsid@ford.com, cdsid@ford.com"
-                        onChange={parseEmails}
-                        data-testid="inviteEditorsFormEmailTextarea"
-                        hidden={collapsed}
-                    />
+                    <label htmlFor="emailTextarea" className="inviteEditorsLabel">
+                    People with this permission can edit
+                        {FEATURE_TOGGLE ?
+                            <Creatable
+                                className="emailTextarea"
+                                inputId="emailTextarea"
+                                styles={inviteEditorsStyle}
+                                placeholder="Enter CDSID of your editors"
+                                menuIsOpen={false}
+                                isMulti={true}
+                                hideSelectedOptions={true}
+                                isClearable={false}
+                                components={components}
+                                hidden={collapsed}
+                                value={invitedUserIds}
+                                options={invitedUserIds}
+                                onChange={onChange}
+                                onInputChange={onInputChange}
+                                inputValue={inputValue}
+                                onKeyDown={handleKeyDownEvent}
```

Comment:
We may want to make this onKeyUp instead

Reply:
Test this: Change onKeyDown to onSubmit, which handles key presses (particularly Enter) automatically.